Packs/Cisco_OpenDNS/Integrations/CiscoOpenDNS/CiscoOpenDNS.py

```diff
@@ -0,0 +1,198 @@
+        r2_j = r2.json()
+        r_code = r2.status_code
+        # r_code ==>
+        # 202 Accepted—Everything worked as expected.
+        # 400 Bad Request—Likely missing a required parameter or malformed JSON. Please check the syntax on your query.
+        # 403 Unauthorized—Request had Authorization header but token was missing or invalid. Please ensure your API token is valid.
+        # 404 Not Found—The requested item doesn't exist, check the syntax of your query or ensure the IP and/or domain are valid. If deleting a domain, ensure the id is correct.
+        # 500, 502, 503, 504 Server errors—Something went wrong on our end.
+        if int(r_code) == 202:
+            ActionResult = "Success"
+            status = True
+        else:
+            ActionResult = "Failed"
+            status = False
+
+        #print("Result :", r2_j)
+        #print("Response Code :", r_code)
+
+        human_readable_data = {
+            'Domain': str(domain),
+            'Action Result Code': r_code,
+            'Response :': r2_j['id'],
+            'ActionResult': ActionResult
+        }
+
+        outputs = {
+            'OpenDNSblockDomain': {
+                'Domain': str(domain),
+                'Action Result Code': r_code,
+                'Response :': r2_j['id'],
+                'ActionResult': ActionResult
+            }
+        }
+
+        headers = ['Domain', 'Action Result Code', 'Response', 'ActionResult']
+        human_readable = tableToMarkdown('OpenDNSblockDomain info', human_readable_data, headers=headers, removeNull=True)
+        return_outputs(human_readable, outputs, r2_j)
+        return status
+    else:
+        human_readable_data = {
+            'Domain': str(domain),
+            'Action Result Code': "Already Blocked",
+            'Response :': "Already Blocked",
+            'ActionResult': "Already Blocked"
+        }
+
+        outputs = {
+            'OpenDNSblockDomain': {
+                'Domain': str(domain),
+                'Action Result Code': "Already Blocked",
+                'Response :': "Already Blocked",
+                'ActionResult': "Already Blocked"
+            }
+        }
+
+        headers = ['Domain', 'Action Result Code', 'Response', 'ActionResult']
+        human_readable = tableToMarkdown('OpenDNSblockDomain info', human_readable_data, headers=headers, removeNull=True)
+        return_outputs(human_readable, outputs)
+        return True
+
+
+def OpenDNSdeleteDomain(myDomainsrequest, myEventsrequest, domain):
+    #print("we are inside del function")
+    delRequest = myDomainsrequest + "&where[name]=" + str(domain)
+    #print("delRequest ",delRequest)
+    r3 = requests.delete(delRequest, timeout=2.50)
+    # r3_txt=r3.json()
+    #print("Response  ", r3_txt)
+    r_code = r3.status_code
+    print("delRequest status ", r_code)
+    if int(r_code) == 204:
+        print(domain + " Domain was removed from blacklist")
```

Comment:
Please use CommandResults to return outputs to warroom or return_results. You can find more information about it [here](https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults)